lib/floe/workflow/state.rb

```diff
@@ -70,7 +70,8 @@ def finish
         context.state["Duration"]       = finished_time - entered_time
         context.execution["EndTime"]    = finished_time_iso if context.next_state.nil?
 
-        logger.info("Running state: [#{context.state_name}] with input [#{context.input}]...Complete - next state: [#{context.next_state}] output: [#{context.output}]")
+        level = context.output&.[]("Error") ? :error : :info
+        logger.send(level, "Running state: [#{self.class.name.split(":").last}:#{context.state_name}] with input [#{context.input}]...Complete - next state: [#{context.next_state || "END"}] output: [#{context.output}]")
```

Comment:
oooh I like the "END" bit - that's nice because it shows it's an end state and not a bug or something.  Not sure about the literal `"END"` - @agrare thoughts?

Reply:
You could have a state named `"END"`, what if instead of `next state: [END]` we left out `next state: []` or said `next state: completed` without the `[]`

Reply:
see changes above

Reply:
For the state_type:state_name change...can we move that duplicated code into a helper method? (ie.. the `#{self.class.name.split(":").last}:#{context.state_name}`